client/src/components/Dialogs/CartModal.js

```diff
@@ -122,113 +131,173 @@ const CartModal = () => {
+                        src={recipe.recipePictureUrl || plate}
+                      />
+                    </Grid>
+                    <Grid
+                      item
+                      container
+                      direction="column"
+                      justify="space-around"
+                      alignItems="flex-start"
+                      xs={6}
+                      spacing={1}
+                    >
+                      <Grid item>
+                        <Typography>{recipe.name}</Typography>
+                      </Grid>
+                      <Grid item>
+                        <span>{qty}</span>{" "}
+                        <ButtonGroup
+                          disableElevation
+                          variant="contained"
+                          className={classes.btnGroup}
+                        >
+                          <Button
+                            onClick={() =>
+                              handleDecrease(recipe._id, qty, recipe.user)
+                            }
+                          >
+                            -
+                          </Button>
+                          <Button
+                            onClick={() =>
+                              handleIncrease(recipe._id, qty, recipe.user)
+                            }
                           >
-                            <Button
-                              onClick={() =>
-                                handleDecrease(recipe._id, qty, recipe.user)
-                              }
-                            >
-                              -
-                            </Button>
-                            <Button
-                              onClick={() =>
-                                handleIncrease(recipe._id, qty, recipe.user)
-                              }
-                            >
-                              +
-                            </Button>
-                          </ButtonGroup>
-                        </TableCell>
-                      </Hidden>
-                      <Hidden smDown>
-                        <TableCell align="right" nowrap="nowrap">{`${
-                          recipe.price
-                        } x ${qty} = ${addDecimals(
-                          qty * recipe.price
-                        )}`}</TableCell>
-                      </Hidden>
-                      <TableCell
-                        align="right"
+                            +
+                          </Button>
+                        </ButtonGroup>
+                      </Grid>
+                    </Grid>
+                    <Grid
+                      item
+                      container
+                      direction="column"
+                      justify="space-around"
+                      alignItems="flex-start"
+                      spacing={2}
+                      xs={3}
+                    >
+                      <Grid item>$ {recipe.price}</Grid>
```

Comment:
It might be a good idea to make this price per unit text smaller or maybe grey. Just an indication that it's for reference and the other price is what is being summed up